File: pinyougou-parent/pinyougou-web-shop/src/main/webapp/js/service/orderService.js
//服务层
app.service('orderService',function($http){

	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post('../order/search.do?page='+page+"&rows="+rows, searchEntity);
	}
	//findOrderAndOrderItem
    this.findOrderAndOrderItem=function(){
        return $http.post('../order/findOrderAndOrderItem.do');
    }
    this.aaa=function(orderId){
        return $http.post('../order/fahuo.do?orderId='+orderId);
    }
});

File: pinyougou-parent/pinyougou-interface/src/main/java/cn/itcast/core/service/OrderService.java
package cn.itcast.core.service;

import cn.itcast.core.pojo.order.Order;
import entity.PageResult;
import vo.OrderAndOrderItemVo;
import vo.SellerOrderVo;

import java.util.List;
import java.util.Map;

public interface OrderService {
    void add(Order order);

    PageResult search(Integer page, Integer rows, Order order);

    List<OrderAndOrderItemVo> findOrderAndOrderItem(String name);

    List<SellerOrderVo> findSellerOrder();
}
